version 0.13.1 with fixes for drqng CRAN checks

s3alfisc opened the pull request.

s3alfisc committed:
version 0.13.1 with fixes for drqng CRAN checks

s3alfisc committed:
Merge branch 'master' into fix-dqrng-upstream

rstub commented on tests/testthat/test_samplers.R:
```diff
@@ -1,5 +1,9 @@
+
+  set.seed(123123)
+  dqrng::dqRNGkind("Xoroshiro128+")
+  dqrng::dqset.seed(123123)
```
I don't think the above lines are necessary. I had included them in my reprex in order to be sure that the difference in confidence intervals is larger than 0.005. However, I think it is in principle a good idea to run this kind of tests w/o fixing the seed or RNG.

zeileis commented:
@s3alfisc and @rstub thanks for all your work on the `fwildclusterboot` and `dqrng` updates.

Are there any news regarding the CRAN update? I'm also asking because of: https://github.com/cran-task-views/Econometrics/issues/25

rstub commented:
AFAIK one problem is that there has been a lot of development here on GH that has not been published to CRAN yet. So fixing only this thing for CRAN is a bit complex, but not impossible.
@s3alfisc Please let me know if I can be of any help here.

zeileis commented:
Alexander @s3alfisc if it is hard to release the current development version, then please consider backporting the essential fixes to the current CRAN version and release that. If I can help with anything, let me know.

rstub commented:
In [rstub/fwildclusterboot@dqrng-fix](https://github.com/rstub/fwildclusterboot/tree/dqrng-fix?rgh-link-date=2024-08-05T13%3A28%3A33Z) I have put in the minimal fix that should help getting this back unto CRAN. AFAIK we would need a branch based on the v0.13.0 tag to make a PR from this.

zeileis commented:
Great, sounds like a good plan!

s3alfisc commented:
Hi both, thanks for nudging me to get the package back on CRAN! I tried at the end of my 2 week grace period but failed (which was frustrating) and then I got some not-so encouraging feedback on top. I have now addressed all comments & the changes suggested by @rstub after reverting to `0.13.0` in this PR https://github.com/s3alfisc/fwildclusterboot/pull/153 and will now run checks on rhub and win devel. Will then release if both pass, hopefully today =)

rstub commented:
You will have to wait until 2024-08-16, since CRAN is on holiday right now.

s3alfisc commented:
Ah, thanks! 😅😀 so still some days to fix all these broken URL's that have been found in the checks 😀

zeileis commented:
Alexander @s3alfisc, another nudge... :wave:

s3alfisc commented:
Just send out a new round of tests to win develop. Package is already back on r-Universe (since yesterday). Thanks for the reminder @zeileis!
